.github/workflows/ci_cd.yml

```diff
@@ -337,6 +350,26 @@ jobs:
           name: ing_logs
           path: ingestion_server/test/ingestion_logs.txt
 
+  # This job runs when `test-ing` doesn't and always passes, thus allowing
+  # PRs to meet the required checks criteria and be merged.
+  bypass-test-ing:
+    name: Run tests for ingestion-server
+    if: |
+      !cancelled() &&
+      needs.test-ing.result == 'skipped'
+    runs-on: ubuntu-latest
+    needs:
+      - test-ing
+    strategy:
+      matrix:
+        name:
+          - es_current
+          - es_7
+
+    steps:
+      - name: Pass
+        run: echo 'Ingestion server tests are skipped because ingestion server is unchanged.'
```

Comment:
Do we really need to update this step since it's not really doing anything? It's just the "Pass" step.